File: frontend/heartmage-candles/src/typesV2/order/BasketItem.ts
import { ConfiguredCandle } from './ConfiguredCandle';
import { ConfiguredCandleFilter } from './ConfiguredCandleFilter';

export interface BasketItem {
  configuredCandle: ConfiguredCandle;
  price: number;
  quantity: number;
  configuredCandleFilter: ConfiguredCandleFilter;
}

File: backend/HeartmadeCandles/HeartmadeCandles.Bot.Core/Models/BasketItem.cs
namespace HeartmadeCandles.Bot.Core.Models;

public class BasketItem
{
    public required ConfiguredCandle ConfiguredCandle { get; init; }

    public decimal Price { get; init; }

    public int Quantity { get; init; }

    public string? ConfiguredCandleFilterString { get; init; }
}
